### PR TITLE
Projects to grid

### PR DESCRIPTION
**Pull request by tul1**
this PR comes from the following issue:
https://github.com/ekumenlabs/ekumenlabs.github.io/issues/151

### CODE REVIEW

**Comment by tul1** [comment]
@german-e-mas need review to merge.

**Comment by tul1** [comment]
@german-e-mas need review.

File: source/stylesheets/projects.scss [hunk 1]
```diff
@@ -23,17 +23,22 @@
...
-
-  .projects-sidebar {
-    .active > a {
-      font-weight: bold;
-    }
-
-    .nav {
-      width: $grid-gutter-width * 8;
-    }
+  section {
```
Comment (german-e-mas) [inline source/stylesheets/projects.scss]: nit: Add a new line before each selector.

File: source/stylesheets/projects.scss [hunk 2]
```diff
@@ -23,17 +23,22 @@
...
+    margin: 0.5rem;
+    padding-right: 1rem;
+    padding-left: 1rem;
+    width: 35rem;
```
Comment (german-e-mas) [inline source/stylesheets/projects.scss]: Have you made sure this is responsive?

Comment (tul1) [reply source/stylesheets/projects.scss]: Yes, it is. However we need to change the content of the project description  so every description has a similar number of words.

0 0 0 0_4567_projects_ iphone 6_7_8

**Comment by tul1** [comment]
@german-e-mas I added your review and a small feature to make it more responsive.
Let me know if I can merge it.

**Comment by german-e-mas** [comment]
@tul1 I don't see any new commits. Have you pushed them?

File: source/projects.html.erb [hunk 3]
```diff
@@ -6,20 +6,18 @@ js:
...
 ---
 
-<div class="row">
-  <div class="col-md-9">
-    <article id="projects">
-      <h1>Projects</h1>
+<div class="projects">
+  <article id="projects">
+    <h1>Projects</h1>
+    <div class="articules">
```
Comment (german-e-mas) [inline source/projects.html.erb]: Please check my previous reviews. Typo: Articles

File: source/projects.html.erb [hunk 4]
```diff
@@ -68,55 +60,28 @@ js:
...
+        $max_projects = articles.count if articles.count > $max_projects
+      end
+    %>
+    <% blog.tags.each do |tag, articles| %>
+      <span class="label label-info" style="background: rgba(73, 169, 222,<%= Math.sqrt(articles.count / $max_projects.to_f)%>)">
+        <%= link_to "#{tag} (#{articles.count})", tag_path(tag) %></span>
+    <% end %>
   </div>
-</div>
+</div>
```
Comment (german-e-mas) [inline source/projects.html.erb]: All files must end with a new line. Add one here.

File: source/stylesheets/projects.scss [hunk 5]
```diff
@@ -23,17 +23,25 @@
...
+    @media (min-width: 991px) {
+      margin: 0.5rem;
+      padding-right: 1rem;
+      padding-left: 1rem;
+      width: 35rem;
+      height: 37.5rem;
     }
+    text-align: justify;
+  }
+  section:hover {
```
Comment (german-e-mas) [inline source/stylesheets/projects.scss]: New line before this selector.

File: source/stylesheets/projects.scss [hunk 6]
```diff
@@ -23,17 +23,25 @@
...
+    background: $gray;
+  }
+  .articules {
```
Comment (german-e-mas) [inline source/stylesheets/projects.scss]: Typo: Articles.
Add a new line before this selector.

File: source/stylesheets/projects.scss [hunk 7]
```diff
@@ -23,17 +23,25 @@
...
+    display:flex;
```
Comment (german-e-mas) [inline source/stylesheets/projects.scss]: Add space between property and value.

**Comment by tul1** [comment]
@german-e-mas sorry, now the code it is pushed.

File: source/stylesheets/projects.scss [hunk 8]
```diff
@@ -23,17 +23,25 @@
...
 
-  .projects-sidebar {
-    .active > a {
-      font-weight: bold;
-    }
-
-    .nav {
-      width: $grid-gutter-width * 8;
+  section {
+    @media (min-width: 991px) {
```
Comment (german-e-mas) [inline source/stylesheets/projects.scss]: Why exactly `991px`? I suggest to change this to `959px` as it is in line of the [breakpoints of Materal Design](https://material.io/design/layout/responsive-layout-grid.html#breakpoints)

Comment (tul1) [reply source/stylesheets/projects.scss]: 991px is the difference beteween bootstrap sm and xs.

File: source/stylesheets/projects.scss (hunk 7 above)
Comment (tul1) [reply source/stylesheets/projects.scss]: what do you mean with 'value' @german-e-mas ?

Comment (german-e-mas) [reply source/stylesheets/projects.scss]: The value is `flex` in this case. The structure is `property: value`. There should be a space between them.

**Comment by tul1** [comment]
Allright @german-e-mas I think I'm ready for a new review. I added all your previous comments.
By the way, thanks for them :-).

File: source/stylesheets/projects.scss [hunk 9]
```diff
@@ -23,17 +23,27 @@
...
+  section:hover {
+    background: $gray;
+  }
 
-    .nav {
-      width: $grid-gutter-width * 8;
-    }
+  .articles {
+    display:flex; 
+    justify-content: space-between;
```
Comment (german-e-mas) [inline source/stylesheets/projects.scss]: Use `space-around`, as it works better with devices that show only two columns.

| space-between | space-around |
| --- | --- |
| localhost_4567_projects_ 4 | localhost_4567_projects_ 3 |

Comment (tul1) [reply source/stylesheets/projects.scss]: done! I also removed `text-align: justify` from the articles titles.

**Comment by tul1** [comment]
@german-e-mas need a new review.

**Comment by german-e-mas** [comment]
LGTM!

**Pull request merged**